js/ui/windowManager.js

```diff
@@ -2375,8 +2692,8 @@ var WindowManager = class {
                     this._showOtherWindows(actor, false);
             }
 
-            /* If this is the Discovery Feed, notify it that it has
-             * finished closing now */
+            // If this is the Discovery Feed, notify it that it has
+            // finished closing now
```

Comment:
Could you split this out, and mention which commit this should be squashed with for the next rebase?

Reply:
done, I've added a new commit with a comment for the future.